Sources/RichTextKit/Component/RichTextViewComponent+Pasting.swift

```diff
@@ -70,12 +70,10 @@ public extension RichTextViewComponent {
         let isSelectedRange = (index == selectedRange.location)
         if isSelectedRange { deleteCharacters(in: selectedRange) }
         if move { moveInputCursor(to: index) }
-        let fontSize = currentFontSize
         images.reversed().forEach { performPasteImage($0, at: index) }
         if move { moveInputCursor(to: safeInsertRange.location + items) }
-        if move || isSelectedRange, let fontSize {
+        if move || isSelectedRange {
             DispatchQueue.main.async {
-                self.setRichTextFontSize(fontSize)
```

Comment:
This was added to try to fix that the font size became 12 after pasting an image. Would be wonderful if the font fix above fixed that.